src/decisionengine/framework/logicengine/LogicEngine.py

```diff
@@ -10,7 +10,8 @@
 class LogicEngine(Module):
     def __init__(self, cfg):
         super().__init__(cfg)
-        self.logger = logging.getLogger()
+        self.logger = structlog.getLogger()
```

Comment:
Should this be grabbing a specific name?

Reply:
Yes, I will change that.